eva/parser/utils.py

```diff
@@ -39,6 +40,23 @@ def parse_table_clause(expr: str):
     return stmt.from_table
 
 
+def parse_create_udf(
+    udf_name: str, implementation: str, type: str, task: str, model: str, **kwargs
+):
+    mock_query = f"CREATE UDF IF NOT EXISTS {udf_name}"
+    if type is not None:
+        mock_query += f" TYPE {type}"
+        if task is not None and model is not None:
+            mock_query += f" 'task' '{task}' 'model' '{model}'"
+    else:
+        mock_query += f" IMPL '{implementation}'"
+    mock_query += ";"
+
+    stmt = Parser().parse(mock_query)[0]
```

Comment:
We need to take care of constructing Abstract Expressions etc. Plus, code maintenance.

Reply:
It can be nontrivial to construct the original query if we have a very long .calls. For an unrealistic example,  .project().filter().apply().grouby().orderby().apply().project().filter().join()....

Reply:
The current design is not constructing the entire query at once. It is rather incrementally constructing the query. I think it should be able to take care of it. The idea is quite similar to constructing the plan but I wanted to reuse the binder,  so went with this design. Do you think of any obvious issue?

Reply:
I think you meant reuse parser. I guess the benefit is that we can do `print(mnist_query.sql_query())` which prints out the SQL query.